### DISCUSSION TITLE
Legacy x58 & i7 920 compatibility

### DISCUSSION THREAD

**Discussion by pratappal** [sebanc/brunch#1664] [Q&A]
Hello need some guidance please -

Does a legacy x58 board with 1st gen intel core i7 support brunch steps to install chrome OS either single or dual boot?
i've tried dual boot instruction step by step (rammus 105, brunch r103 & MBR patch) for both - with ubuntu or with win10 but no luck! 

with ubuntu it fails to detect chromeos.img and kernel loading during boot
with win0 it does not ask "dualboot" confirmation - also fails to generate chromeos.grub.txt

please help with any possible guidance.

Thanks

**Comment by tejasraman** [comment]
You cannot use any ChromeOS newer than 81 with that CPU. You could try ChromeOS flex though (this is my "canned solution" for unsupported CPU users)

**Reply by kocoman1** [reply]
can you tell me what is changed in 81? need some new sse4.2 or avx? thx
